### PR TITLE
coreclr: Automatic port of triple slash from System.Threading.Tasks.Sources

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners: @tarekgh, @stephentoub
Language owners: @mairaw @rpetrusha

### CODE REVIEW

**Comment by mairaw** [comment]
Can you replace the `&lt;` and `&gt;` with the actual < and > @carlossanlop?

**Comment by carlossanlop** [comment]
@mairaw oops let me fix that real quick. It was supposed to be converted automatically.

**Comment by carlossanlop** [comment]
@mairaw Fixed!

File: xml/System.Threading.Tasks.Sources/IValueTaskSource.xml [hunk 1]
```diff
@@ -114,11 +114,11 @@
         <Parameter Name="flags" Type="System.Threading.Tasks.Sources.ValueTaskSourceOnCompletedFlags" />
       </Parameters>
       <Docs>
-        <param name="continuation">To be added.</param>
-        <param name="state">To be added.</param>
-        <param name="token">To be added.</param>
-        <param name="flags">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="continuation">The continuation to invoke when the operation has completed.</param>
+        <param name="state">The state object to pass to &lt;paramref name="continuation" /> when it's invoked.</param>
```
Comment (stephentoub) [inline xml/System.Threading.Tasks.Sources/IValueTaskSource.xml]: Is this &lt; correct when the paramref then ends with /> ?

File: xml/System.Threading.Tasks.Sources/IValueTaskSource`1.xml [hunk 2]
```diff
@@ -122,11 +122,11 @@
         <Parameter Name="flags" Type="System.Threading.Tasks.Sources.ValueTaskSourceOnCompletedFlags" />
       </Parameters>
       <Docs>
-        <param name="continuation">To be added.</param>
-        <param name="state">To be added.</param>
-        <param name="token">To be added.</param>
-        <param name="flags">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="continuation">The continuation to invoke when the operation has completed.</param>
+        <param name="state">The state object to pass to &lt;paramref name="continuation" /> when it's invoked.</param>
```
Comment (stephentoub) [inline xml/System.Threading.Tasks.Sources/IValueTaskSource`1.xml]: Same question.  Will stop asking.

File: xml/System.Threading.Tasks.Sources/ManualResetValueTaskSourceCore`1.xml [hunk 3]
```diff
@@ -192,7 +194,7 @@
       </Parameters>
       <Docs>
         <param name="error">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Complets with an error.</summary>
```
Comment (stephentoub) [inline xml/System.Threading.Tasks.Sources/ManualResetValueTaskSourceCore`1.xml]: Complets => Completes

File: xml/System.Threading.Tasks.Sources/IValueTaskSource.xml (hunk 1 above)
Comment (rpetrusha) [reply xml/System.Threading.Tasks.Sources/IValueTaskSource.xml]: No, it's not correct, @stephentoub. I've changed it.

**Pull request merged**